spec/api/v1/rolls_request_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+
+describe "Roller API" do
+
+  before(:each) do
+    @roll = Roll.create(title: 'Rick Roll', start_time: 19, end_time: 23, youtube_indentifer: 'lYbYoaBrXpU')
```

Comment:
Not very important, but can you change the rolls to match the ones from the model spec? I changed the title and added another one that you can use for the second one (in place of fenton roll).